site/gdocs/Chart.tsx

```diff
@@ -42,3 +89,57 @@ export default function Chart({
         </div>
     )
 }
+
+const mapKeywordToGrapherConfig = (
```

Comment:
yeah I wanted it to be forgiving but I think you're right that this will probably lead to more confusion / headache in the future... I'll remove them!

Reply:
I agree with Christian here - I think the ambiguity here is not particularly helpful. In the end, this is a controlled vocabulary that matches onto something pretty technical. At the same time, the fields that we want to influence have a different origin story and so some are called hide* and some are called has* etc.

We use this inside a key called `show` - as such I'd be in favour of using a camelCase fragment that matches the config field this relates to. This should make it straightforward to figure out the field this actually influences. Authors are exposed to these names already through tools like the bulk chart editor.

So I'd suggest names like
```
relativeToggle
timeline
facetControl
chartTab
```

Reply:
Ah you posted your comment just as I had finished writing it as part of my review :)

Reply:
haha this is all a bit confusing

Good point that authors are exposed to these names anyway. I'll camelCase them!